react-app/ccf/src/pages/applicant_results/ResultsPage.css

```
@@ -0,0 +1,136 @@
+}
+
+.review-section{        
+    margin-bottom: 0.5rem;    
+}
+
+.review-text{
+    font-size: 0.85rem;
+    font-weight: 600;        
+    max-width: 100%;
+    margin-bottom: 1rem; 
+    font-weight: 400;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-weight is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)